ReactWindows/ReactNative.Shared.Tests/Modules/WebSocket/WebSocketModuleTests.cs

```diff
@@ -62,15 +63,22 @@ public void WebSocketModule_FailedEvent()
         {
             var waitHandle = new AutoResetEvent(false);
             var json = default(JObject);
+            bool websocketFailedCalled = false;
+
```

Comment:
Take a look at a bunch of the comments from the ScrollView review, same stuff needs to be cleaned up here.